amnestywebsite/humanity-theme
# Changes Follow Us On translation for AR social media icons

Epic: https://github.com/amnestywebsite/humanity-theme/issues/417
Issue: https://github.com/amnestywebsite/amnesty-wp-theme/issues/2973

**Steps to test**:
1. Navigate to /ar/
2. Scroll to the site footer and look for the social icons
3. Check the "Follow us on" translation, which should now be تابعونا على:

**Considerations**:
- Localisation

Example: http://bigbite.im/i/7WXUjL
